doc/source/tools/libraries/index.rst

```diff
@@ -10,7 +10,7 @@ Tooling libraries
       :text-align: center
       :class-title: pyansys-card-title
 
-      Simple gRPC API tool for moving files between a client and a remote server
+      Simple gRPC API tool for moving files between a client and a remote server.
```

Comment:
Revert the points at the end of the phrases. We are avoiding them on purpose due to the way they render in cards

Reply:
Some of them had them and some didn't, I'll remove all

Reply:
Yeah the ones that had it, were probably my fault. Sorry.